torch/csrc/jit/mobile/backport_manager.cpp

```diff
@@ -349,6 +352,95 @@ std::stringstream backport_v6_to_v5(std::stringstream& input_model_stream) {
 
   return ouput_model_stream;
 }
+
+std::stringstream backport_v7_to_v6(std::stringstream& input_model_stream) {
+  std::shared_ptr<IStreamAdapter> rai =
+      std::make_shared<IStreamAdapter>(&input_model_stream);
+  auto reader = std::make_shared<PyTorchStreamReader>(rai);
```

Comment:
Just nit: The initial part of this method seems similar to backport_v6_to_v5. Should we make it into an utility function?

Reply:
Good point! Resolve in the latest commit. Create a function to re-emit the model, and the input is model_stream + different mode.